Source/RxSwift/Observable+Moya.swift

```diff
@@ -1,5 +1,8 @@
 import Foundation
 import RxSwift
+#if !COCOAPODS
```

Comment:
I think we should use our own swift flag instead of piggybacking on the cocoapods one. Something like `IMPORTS_MOYA` or something.